wire-ios/Tests/Mocks/UserSessionMock.swift

```diff
@@ -252,6 +252,10 @@ final class UserSessionMock: UserSession {
         mockGetUserClientFingerprintUseCaseProtocol
     }
 
+    var getSelfUserVerificationStatusUseCase: GetSelfUserVerificationStatusUseCaseProtocol {
+        fatalError("not implemented yet")
```

Comment:
Will you implement it in this PR?

Reply:
It's probably not needed. I will change the description.